src/Microsoft.AspNetCore.Razor.Design/build/netstandard2.0/Microsoft.AspNetCore.Razor.Design.props

```diff
@@ -11,8 +11,8 @@
     Use these properties to attach behavior to the corresponding target.
   -->
   <PropertyGroup>
-    <RazorGenerateDependsOn>_RazorResolveTagHelpers;RazorCoreGenerate</RazorGenerateDependsOn>
-    <RazorCompileDependsOn>_RazorResolveTagHelpers;RazorCoreGenerate;RazorCoreCompile</RazorCompileDependsOn>
+    <RazorGenerateDependsOn>RazorResolveGenerateInputs;RazorCoreGenerate</RazorGenerateDependsOn>
```

Comment:
This needs to be a separate entry in the DependsOn chain. It's not enough to have RazorCoreGenerate depend on it, because it won't be evaluated soon enough.